packages/cli/src/WaitTracker.ts

```diff
@@ -56,7 +56,7 @@ export class WaitTrackerClass {
 		const findQuery: FindManyOptions<IExecutionFlattedDb> = {
 			select: ['id', 'waitTill'],
 			where: {
-				waitTill: LessThanOrEqual(new Date(Date.now() + 70000)),
+				waitTill: LessThanOrEqual(new Date(Date.now() + parseInt(`${process.env.N8N_WAIT_CYCLE_INTERVAL ?? 60000}`) + 10000)),
```

Comment:
Is there a reason to not let this sleep duration be below 10s?
Same question for the other occurence of a hardcoded `10000` below

Reply:
I don't really know, it was already hardcoded with `+10000` from the hardcoded original `60000 ms`, but as the comment say, it should be because of an issue with SQLite and TypeORM